csep/core/brier_evaluations.py

```diff
@@ -0,0 +1,156 @@
+import numpy
+import scipy.stats
+import scipy.spatial
+
+from csep.models import EvaluationResult
+from csep.core.exceptions import CSEPCatalogException
+
+
+def bier_score_ndarray(forecast, observations):
+    """ Computes the brier score
+    """
+
+    observations = (observations >= 1).astype(int)
+    prob_success = 1 - scipy.stats.poisson.cdf(0, forecast)
+    brier = []
+
+    for p, o in zip(prob_success.ravel(), observations.ravel()):
+
+        if o == 0:
+            brier.append(-2 * p ** 2)
+        else:
+            brier.append(-2 * (p - 1) ** 2)
+    brier = numpy.sum(brier)
+
+    for n_dim in observations.shape:
+        brier /= n_dim
+
+    return brier
+
+
+def _simulate_catalog(sim_cells, sampling_weights, sim_fore, random_numbers=None):
```

Comment:
Why not reuse or merge with [`core/poisson_evaluations._simulate_catalog()`](https://github.com/SCECcode/pycsep/blob/c6d028f50280edd14efa3b1ebc64733c29963e83/csep/core/poisson_evaluations.py#L580)? Or even outsource it?